sources/packages/web/src/components/common/students/StudentProfileLegacyMatches.vue

```diff
@@ -0,0 +1,200 @@
+          propertyTitle="Last name"
+          :propertyValue="legacyProfile.lastName"
+        />
+      </v-col>
+      <v-col>
+        <title-value
+          propertyTitle="Date of birth"
+          :propertyValue="dateOnlyLongString(legacyProfile.dateOfBirth)"
+      /></v-col>
+    </v-row>
+    <v-row>
+      <v-col
+        ><title-value
+          propertyTitle="SIN"
+          :propertyValue="sinDisplayFormat(legacyProfile.sin)"
+        />
+      </v-col>
+    </v-row>
+    <v-row v-if="legacyProfile.hasMultipleProfiles">
+      <v-col
+        ><banner
+          :type="BannerTypes.Warning"
+          summary="The student is associated with multiple legacy profiles; the most recent update is shown."
+        ></banner>
+      </v-col>
+    </v-row>
+  </template>
+  <template v-else>
+    <banner
+      :type="BannerTypes.Info"
+      summary="The student is not currently associated with any legacy record. Please check the legacy match information to see if there are any potential matches."
+    >
+      <template #actions>
+        <v-btn color="primary" @click="loadStudentLegacyMatches()">
+          Check for legacy matches
+          <v-tooltip activator="parent" location="start"
+            >Click to check for potential legacy matches.</v-tooltip
+          >
+        </v-btn></template
+      >
+    </banner>
+    <toggle-content
+      v-if="showLegacyMatches"
+      :toggled="!legacyStudentMatches?.length && !studentLegacyMatchesLoading"
+      message="No potential legacy matches found."
+    >
+      <v-data-table
+        :headers="StudentProfileLegacyMatchHeaders"
+        :items="legacyStudentMatches"
+        :loading="studentLegacyMatchesLoading"
+        :items-per-page="-1"
+      >
+        <template v-slot:loading>
+          <v-skeleton-loader type="table-row@3"></v-skeleton-loader>
+        </template>
+        <template #[`item.firstName`]="{ item }">
+          {{ emptyStringFiller(item.firstName) }}
+        </template>
+        <template #[`item.lastName`]="{ item }">
+          {{ item.lastName }}
+        </template>
+        <template #[`item.birthDate`]="{ item }">
+          {{ dateOnlyLongString(item.birthDate) }}
+        </template>
+        <template #[`item.sin`]="{ item }">
+          {{ sinDisplayFormat(item.sin) }}
+        </template>
+        <template #[`item.actions`]="{ item }">
+          <check-permission-role :role="Role.StudentLinkLegacyProfile">
+            <template #="{ notAllowed }">
+              <v-btn
+                color="primary"
+                :disabled="notAllowed"
+                @click="showLinkProfileModal(item.individualId)"
+              >
+                Link
+                <v-tooltip activator="parent" location="start"
+                  >Click to link student with this legacy profile.</v-tooltip
+                >
+              </v-btn>
+            </template>
+          </check-permission-role>
+        </template>
+        <template #bottom></template>
```

Comment:
Do we require this bottom template for any reason?

Reply:
That was the only effective way to not display the paginator.